poc/vdaf_poc/test_utils.py

```diff
@@ -69,18 +71,88 @@ def run_vdaf_test(
         self.assertEqual(agg_result, expected_agg_result)
 
 
+class VdafTestVectorOperationDict(TypedDict):
+    """
+    A description of one of the operations that should be performed with this
+    test vector.
+
+    Attributes:
+        operation: The type of operation to be performed. This is one of
+            "shard", "prep_init", "prep_shares_to_prep", "prep_next",
+            "aggregate", or "unshard".
```

Comment:
"aggregate" corresponds to `agg_init()` followed by `agg_update()`, correct? We should spell this out.